### ISSUE TITLE
images should have alt tags that arent empty (for accessibility)

### ISSUE THREAD

**Issue by eadehemingway** [fack2/ImageGuessing-Game#13]
https://github.com/fack2/ImageGuessing-Game/blob/523b8abe71b044d868bd3d54545a9618b0879f4d/src/Components/Main/index.js#L67
